spec/cases/graph_api_batch_spec.rb

```diff
@@ -406,6 +416,13 @@
               Koala::Facebook::API.new("foo").batch {|batch_api| batch_api.get_object('me') }
             }.to raise_exception(Koala::Facebook::BadFacebookResponse)
```

Comment:
```suggestion
            }.to raise_exception(Koala::Facebook::BadFacebookResponse, /"Facebook returned an empty body"/)
```

Reply:
We have updated the test to check for the error message as well.